test/test_helper.rb

```diff
@@ -4,7 +4,7 @@
 
 class ActiveSupport::TestCase
   # Run tests in parallel with specified workers
-  parallelize(workers: :number_of_processors)
+  parallelize(workers: 1)
```

Comment:
Why this change?

Reply:
Oh right. I ran into issues trying to run the tests on my personal machine because of the parallelization (it's a 6 core i5). Changing to a single thread fixed it and because there are so few tests, I figured that was ok for now.

Reply:
Ah, OK. We should fix those tests so that they're not problematic, but we can do that in a separate PR.

Reply:
Sounds good. Create an issues to capture that here: https://github.com/CovidShield/portal/issues/74